tests/disableHttp/disableHttp.js

```diff
@@ -12,15 +12,19 @@
 'use strict';
 
 const supertest = require('supertest');
+const expect = require('chai').expect;
 const api = supertest(require('../../express').app);
 const constants = require('../../api/v1/constants');
+const tu = require('../testUtils');
 const path = '/v1/users';
-const expect = require('chai').expect;
 
 describe('tests/disableHttp/disableHttp.js, http is disabled >', () => {
-  it('GET is redirected', (done) => {
+  before(() => tu.toggleOverride('requireHttps', true));
+  after(() => tu.toggleOverride('requireHttps', false));
+
+  it('GET is rejected', (done) => {
```

Comment:
I think both the tests are failing here because of 'Authentication Failed' since we are not providing token.
Also path does not specifically has http - does it defaults to http behind the scene?
We should check the error description as well to make sure we are testing the right thing.

Reply:
Great catch, @pallavi2209! Because we were loading the express module BEFORE setting the toggle override, it actually wasn't using the override. I have updated the test so that it loads the express module AFTER setting the toggle override, and also check the error message text.